src/elements/radio-button/radio-button/radio-button.visual.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { html, nothing } from 'lit';
+
+import { describeEach, describeViewports, visualDiffDefault } from '../../core/testing/private.js';
+
+import '../radio-button.js';
+
+const cases = {
+  size: ['m', 's'],
+  checked: [true, false],
+  disabled: [false, true],
+  bold: [false, true],
```

Comment:
I wonder whether we should test the bold case (for all the test files in this PR). this is entirely on the consumers' side (was just an example in chromatic).

Reply:
Makes sense